llvm/lib/Target/SPIRV/SPIRVPrepareFunctions.cpp

```diff
@@ -376,6 +380,7 @@ SPIRVPrepareFunctions::removeAggregateTypesFromSignature(Function *F) {
 
 bool SPIRVPrepareFunctions::runOnModule(Module &M) {
   bool Changed = false;
+
```

Comment:
Do we really need this line?

Reply:
The newline?
No, not really. Removing and landing. Thanks.